CMakeLists.txt

```diff
@@ -962,8 +962,13 @@ if (BUILD_TESTING)
     mark_as_advanced (HDF5_TEST_FHEAP_PASSTHROUGH VOL)
   endif ()
 
-  option (HDF_TEST_EXPRESS "Control testing framework (0-3)" "0")
```

Comment:
HDF_TEST_EXPRESS needed to change from being an option to a STRING. Otherwise, the only options are ON and OFF in CMake